### ISSUE TITLE
Add kontribusi icon on homepage

### ISSUE THREAD

**Issue by mazipan** [kawalcovid19/wargabantuwarga.com#642]
## Description

![Screenshot_20210811-205227_Chrome](https://user-images.githubusercontent.com/7221389/129041573-376d6052-0914-4b21-844d-1279a2658589.jpg)

Link goes to: https://www.indorelawan.org/activity/60e2ed45164da80018b0e246

## Figma

https://www.figma.com/file/XNNtIoFEdFqaXOee83n0oN/WBW?node-id=485:2485

## Tasks

- [ ] Add new icon on homepage
- [ ] Add testing

**Comment by suryavip** [comment]
Hello, first-timer here. Can you clarify whether the link need to be opened on new tab (like existing link on "Cara Berkontribusi") or not?
![image](https://user-images.githubusercontent.com/28729285/129059436-ad65b330-8c1d-47da-8f23-692f5e80a9a9.png)

**Comment by mazipan** [comment]
Unfortunately it already in progress by @hariangr

But yes, in the current PR, there is a unfinished task.
To open it on the new tab.

So I think you can open new issue and PR after #643 being merged @suryavip 
We need to support to open in new tab

**Comment by mazipan** [comment]
Open in #654 ya mas @hariangr
